sphinx_gallery/gen_rst.py

```diff
@@ -231,6 +235,22 @@ def text2string(content):
         return content
 
 
+def extract_thumbnail_number(text):
+    """ Pull out the thumbnail image number specified in the docstring. """
+
+    # check whether the user has specified a specific thumbnail image
+    pattr = re.compile("^#\s*sphinx_gallery_thumbnail_number:\s*([0-9]+)\s*$", flags=re.MULTILINE)
```

Comment:
Hmmm then maybe move the sphinx_gallery_thumbnail_number it somewhere else in the file ;-)